apps/src/applab/designMode.js

```diff
@@ -859,7 +859,7 @@ designMode.renderToggleRow = function (screenIds) {
   var designToggleRow = document.getElementById('designToggleRow');
   if (designToggleRow) {
     React.render(
-      React.createElement(DesignToggleRow, {
+      React.createElement(PlaySpaceHeader, {
```

Comment:
Hah. I bet we can tell our linter about JSX (though it's possible it would need an update)